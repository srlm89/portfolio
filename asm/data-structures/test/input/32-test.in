ERASE actual/32-test.out
INSERT 13 1.281106 Facing The Giants 720p
PRINT_IN_ORDER actual/32-test.out
INSERT 25 2.411041 Bugs.A.Rainforest.Adventure.3D.HD.DVD.REMUX.1080p.VC-1-SEBUL
INSERT 30 2.144422 Lawrence of Arabia   1080p
REMOVE_ID 25
PRINT_IN_ORDER actual/32-test.out
INSERT 24 9.563535 Resident Evil Series    1080p
INSERT 17 6.802900 American Gangster    720p
INSERT 15 1.629359 X-Men 4 - Origins Wolverine (2009)  1080p
REMOVE_ID 24
PRINT_IN_ORDER actual/32-test.out
INSERT 36 5.165468 Starship Troopers 1  1080p
INSERT 21 0.789850 Inkheart [2008] [720p]
INSERT 10 2.246429 The Day After Tomorrow 1080p
INSERT 34 4.713719 Casablana.1942.1080p.BluRay.x264.Codres
REMOVE_ID 21
REMOVE_ID 15
PRINT_IN_ORDER actual/32-test.out
INSERT 26 3.615149 All About Steve 720P
INSERT 9 7.666585 Spiderman 1   1080p
INSERT 1 7.406024 Doomsday 720p
INSERT 20 8.039906 The.Godfather.Part.II.1974.720p.BluRay.x264-ESiR_tehPARADOX.COM
INSERT 2 7.406335 X-Men 1    720p
REMOVE_ID 30
REMOVE_ID 26
PRINT_IN_ORDER actual/32-test.out
INSERT 4 8.696302 Meet The Parents    720p
INSERT 7 7.442432 Beauty.And.The.Beast.1991.720p.HDTV.x264-hV
INSERT 45 6.545009 Final Destination 3   1080p
INSERT 31 8.125237 007.Die Another Day.avi
INSERT 44 1.230578 Scooby.Doo.The.Mystery.Begins.2009   720p
INSERT 41 2.165705 X-Men Origin - Wolverine [2009] [1080p]
REMOVE_ID 17
REMOVE_ID 45
REMOVE_ID 2
PRINT_IN_ORDER actual/32-test.out
INSERT 16 9.652257 Flash Point    720p
INSERT 40 9.282939 Defiance.1080p
INSERT 28 7.497149 New York I Love You 720p   720p
INSERT 29 8.704252 One Missed Call   720p
INSERT 14 8.059184 Epic Movie UNRATED (2007) 720p
INSERT 48 8.746965 The Usual Suspect    720p
INSERT 42 1.732968 Up [1080p]
REMOVE_ID 40
REMOVE_ID 34
REMOVE_ID 31
PRINT_IN_ORDER actual/32-test.out
INSERT 18 7.904498 Lawrence of Arabia   1080p
INSERT 0 4.891536 Mission Impossible Trilogy   720p
INSERT 59 4.803718 Kill Bill Vol 2 [2004]   720p
INSERT 58 5.492698 High school musical II 2007    720p
INSERT 3 5.163157 Daredevil - 720p
INSERT 64 7.530775 Baby on Board (2009)    720p
INSERT 49 5.450318 8 Mile [2002] [720p]
INSERT 54 1.215715 Deep Sea
REMOVE_ID 29
REMOVE_ID 14
REMOVE_ID 64
REMOVE_ID 28
PRINT_IN_ORDER actual/32-test.out
INSERT 35 2.220425 Bolt [2008] [1080p]
INSERT 65 0.590498 Crossing Over 720P
INSERT 69 7.154971 Night At The Museum 2 [720p]
INSERT 53 0.633442 Rocky 3 [1982] [BluRay] [720p] [AC3]
INSERT 55 3.114966 Beauty And The Beast    720p
INSERT 52 2.445253 007 The Man With the Golden Gun [1974] [BluRay] [720p] [DTS]
INSERT 71 2.696676 Elektra 720p
INSERT 5 0.531011 Pirates of Caribbean 1-2-3 Collection   1080p
INSERT 23 1.722579 The Bank Job [720p]
REMOVE_ID 49
REMOVE_ID 5
REMOVE_ID 23
REMOVE_ID 54
PRINT_IN_ORDER actual/32-test.out
INSERT 73 6.929544 Underworld Series
INSERT 68 0.310098 Oxford Murders, The.720p
INSERT 8 3.150418 Daredevil - 720p
INSERT 80 5.072942 PBS
INSERT 78 1.601014 Lord Of The Ring 1 - Fellowship of the Ring (ExtEd) [1080p]
INSERT 46 7.522239 Beowulf   720p
INSERT 37 1.047873 Death Sentence   1080p
INSERT 60 5.887429 The.Terminal.2004.720p.HDTV.x264-ESiR
INSERT 6 5.168002 Push 720P
INSERT 79 4.030397 Equilibrium 720p
REMOVE_ID 52
REMOVE_ID 80
REMOVE_ID 41
REMOVE_ID 37
REMOVE_ID 3
PRINT_IN_ORDER actual/32-test.out
INSERT 32 7.323695 Fearless - 720p
INSERT 22 5.115085 Before The Devil Knows, You're Dead   1080p
INSERT 47 5.277156 Apocalypto   720p
INSERT 89 3.984421 The.Godfather.Part.III.1990.720p.BluRay.x264-SiNNERS_tehPARADOX.COM
INSERT 87 8.462529 Back.to.the.Future.Part.III
INSERT 90 5.320547 James Bond 09 - The Man with the Golden Gun [1974]
INSERT 61 0.577450 The Orphan [720]
INSERT 82 6.525550 Armageddon [1998] [720p]
INSERT 56 2.815446 Get Smart (2008) 1080p
INSERT 91 7.080348 Indiana.Jones.and.the.Kingdom.of.the.Crystal.Skull    1080p
INSERT 74 6.336581 The Lion King [1994] [720p]
REMOVE_ID 68
REMOVE_ID 55
REMOVE_ID 56
REMOVE_ID 44
REMOVE_ID 48
PRINT_IN_ORDER actual/32-test.out
