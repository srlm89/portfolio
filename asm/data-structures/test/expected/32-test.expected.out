[ 13 1.281106 Facing The Giants 720p ]
[ 13 1.281106 Facing The Giants 720p ][ 30 2.144422 Lawrence of Arabia   1080p ]
[ 13 1.281106 Facing The Giants 720p ][ 15 1.629359 X-Men 4 - Origins Wolverine (2009)  1080p ][ 17 6.802900 American Gangster    720p ][ 30 2.144422 Lawrence of Arabia   1080p ]
[ 10 2.246429 The Day After Tomorrow 1080p ][ 13 1.281106 Facing The Giants 720p ][ 17 6.802900 American Gangster    720p ][ 30 2.144422 Lawrence of Arabia   1080p ][ 34 4.713719 Casablana.1942.1080p.BluRay.x264.Codres ][ 36 5.165468 Starship Troopers 1  1080p ]
[ 1 7.406024 Doomsday 720p ][ 2 7.406335 X-Men 1    720p ][ 9 7.666585 Spiderman 1   1080p ][ 10 2.246429 The Day After Tomorrow 1080p ][ 13 1.281106 Facing The Giants 720p ][ 17 6.802900 American Gangster    720p ][ 20 8.039906 The.Godfather.Part.II.1974.720p.BluRay.x264-ESiR_tehPARADOX.COM ][ 34 4.713719 Casablana.1942.1080p.BluRay.x264.Codres ][ 36 5.165468 Starship Troopers 1  1080p ]
[ 1 7.406024 Doomsday 720p ][ 4 8.696302 Meet The Parents    720p ][ 7 7.442432 Beauty.And.The.Beast.1991.720p.HDTV.x264-hV ][ 9 7.666585 Spiderman 1   1080p ][ 10 2.246429 The Day After Tomorrow 1080p ][ 13 1.281106 Facing The Giants 720p ][ 20 8.039906 The.Godfather.Part.II.1974.720p.BluRay.x264-ESiR_tehPARADOX.COM ][ 31 8.125237 007.Die Another Day.avi ][ 34 4.713719 Casablana.1942.1080p.BluRay.x264.Codres ][ 36 5.165468 Starship Troopers 1  1080p ][ 41 2.165705 X-Men Origin - Wolverine [2009] [1080p] ][ 44 1.230578 Scooby.Doo.The.Mystery.Begins.2009   720p ]
[ 1 7.406024 Doomsday 720p ][ 4 8.696302 Meet The Parents    720p ][ 7 7.442432 Beauty.And.The.Beast.1991.720p.HDTV.x264-hV ][ 9 7.666585 Spiderman 1   1080p ][ 10 2.246429 The Day After Tomorrow 1080p ][ 13 1.281106 Facing The Giants 720p ][ 14 8.059184 Epic Movie UNRATED (2007) 720p ][ 16 9.652257 Flash Point    720p ][ 20 8.039906 The.Godfather.Part.II.1974.720p.BluRay.x264-ESiR_tehPARADOX.COM ][ 28 7.497149 New York I Love You 720p   720p ][ 29 8.704252 One Missed Call   720p ][ 36 5.165468 Starship Troopers 1  1080p ][ 41 2.165705 X-Men Origin - Wolverine [2009] [1080p] ][ 42 1.732968 Up [1080p] ][ 44 1.230578 Scooby.Doo.The.Mystery.Begins.2009   720p ][ 48 8.746965 The Usual Suspect    720p ]
[ 0 4.891536 Mission Impossible Trilogy   720p ][ 1 7.406024 Doomsday 720p ][ 3 5.163157 Daredevil - 720p ][ 4 8.696302 Meet The Parents    720p ][ 7 7.442432 Beauty.And.The.Beast.1991.720p.HDTV.x264-hV ][ 9 7.666585 Spiderman 1   1080p ][ 10 2.246429 The Day After Tomorrow 1080p ][ 13 1.281106 Facing The Giants 720p ][ 16 9.652257 Flash Point    720p ][ 18 7.904498 Lawrence of Arabia   1080p ][ 20 8.039906 The.Godfather.Part.II.1974.720p.BluRay.x264-ESiR_tehPARADOX.COM ][ 36 5.165468 Starship Troopers 1  1080p ][ 41 2.165705 X-Men Origin - Wolverine [2009] [1080p] ][ 42 1.732968 Up [1080p] ][ 44 1.230578 Scooby.Doo.The.Mystery.Begins.2009   720p ][ 48 8.746965 The Usual Suspect    720p ][ 49 5.450318 8 Mile [2002] [720p] ][ 54 1.215715 Deep Sea ][ 58 5.492698 High school musical II 2007    720p ][ 59 4.803718 Kill Bill Vol 2 [2004]   720p ]
[ 0 4.891536 Mission Impossible Trilogy   720p ][ 1 7.406024 Doomsday 720p ][ 3 5.163157 Daredevil - 720p ][ 4 8.696302 Meet The Parents    720p ][ 7 7.442432 Beauty.And.The.Beast.1991.720p.HDTV.x264-hV ][ 9 7.666585 Spiderman 1   1080p ][ 10 2.246429 The Day After Tomorrow 1080p ][ 13 1.281106 Facing The Giants 720p ][ 16 9.652257 Flash Point    720p ][ 18 7.904498 Lawrence of Arabia   1080p ][ 20 8.039906 The.Godfather.Part.II.1974.720p.BluRay.x264-ESiR_tehPARADOX.COM ][ 35 2.220425 Bolt [2008] [1080p] ][ 36 5.165468 Starship Troopers 1  1080p ][ 41 2.165705 X-Men Origin - Wolverine [2009] [1080p] ][ 42 1.732968 Up [1080p] ][ 44 1.230578 Scooby.Doo.The.Mystery.Begins.2009   720p ][ 48 8.746965 The Usual Suspect    720p ][ 52 2.445253 007 The Man With the Golden Gun [1974] [BluRay] [720p] [DTS] ][ 53 0.633442 Rocky 3 [1982] [BluRay] [720p] [AC3] ][ 55 3.114966 Beauty And The Beast    720p ][ 58 5.492698 High school musical II 2007    720p ][ 59 4.803718 Kill Bill Vol 2 [2004]   720p ][ 65 0.590498 Crossing Over 720P ][ 69 7.154971 Night At The Museum 2 [720p] ][ 71 2.696676 Elektra 720p ]
[ 0 4.891536 Mission Impossible Trilogy   720p ][ 1 7.406024 Doomsday 720p ][ 4 8.696302 Meet The Parents    720p ][ 6 5.168002 Push 720P ][ 7 7.442432 Beauty.And.The.Beast.1991.720p.HDTV.x264-hV ][ 8 3.150418 Daredevil - 720p ][ 9 7.666585 Spiderman 1   1080p ][ 10 2.246429 The Day After Tomorrow 1080p ][ 13 1.281106 Facing The Giants 720p ][ 16 9.652257 Flash Point    720p ][ 18 7.904498 Lawrence of Arabia   1080p ][ 20 8.039906 The.Godfather.Part.II.1974.720p.BluRay.x264-ESiR_tehPARADOX.COM ][ 35 2.220425 Bolt [2008] [1080p] ][ 36 5.165468 Starship Troopers 1  1080p ][ 42 1.732968 Up [1080p] ][ 44 1.230578 Scooby.Doo.The.Mystery.Begins.2009   720p ][ 46 7.522239 Beowulf   720p ][ 48 8.746965 The Usual Suspect    720p ][ 53 0.633442 Rocky 3 [1982] [BluRay] [720p] [AC3] ][ 55 3.114966 Beauty And The Beast    720p ][ 58 5.492698 High school musical II 2007    720p ][ 59 4.803718 Kill Bill Vol 2 [2004]   720p ][ 60 5.887429 The.Terminal.2004.720p.HDTV.x264-ESiR ][ 65 0.590498 Crossing Over 720P ][ 68 0.310098 Oxford Murders, The.720p ][ 69 7.154971 Night At The Museum 2 [720p] ][ 71 2.696676 Elektra 720p ][ 73 6.929544 Underworld Series ][ 78 1.601014 Lord Of The Ring 1 - Fellowship of the Ring (ExtEd) [1080p] ][ 79 4.030397 Equilibrium 720p ]
[ 0 4.891536 Mission Impossible Trilogy   720p ][ 1 7.406024 Doomsday 720p ][ 4 8.696302 Meet The Parents    720p ][ 6 5.168002 Push 720P ][ 7 7.442432 Beauty.And.The.Beast.1991.720p.HDTV.x264-hV ][ 8 3.150418 Daredevil - 720p ][ 9 7.666585 Spiderman 1   1080p ][ 10 2.246429 The Day After Tomorrow 1080p ][ 13 1.281106 Facing The Giants 720p ][ 16 9.652257 Flash Point    720p ][ 18 7.904498 Lawrence of Arabia   1080p ][ 20 8.039906 The.Godfather.Part.II.1974.720p.BluRay.x264-ESiR_tehPARADOX.COM ][ 22 5.115085 Before The Devil Knows, You're Dead   1080p ][ 32 7.323695 Fearless - 720p ][ 35 2.220425 Bolt [2008] [1080p] ][ 36 5.165468 Starship Troopers 1  1080p ][ 42 1.732968 Up [1080p] ][ 46 7.522239 Beowulf   720p ][ 47 5.277156 Apocalypto   720p ][ 53 0.633442 Rocky 3 [1982] [BluRay] [720p] [AC3] ][ 58 5.492698 High school musical II 2007    720p ][ 59 4.803718 Kill Bill Vol 2 [2004]   720p ][ 60 5.887429 The.Terminal.2004.720p.HDTV.x264-ESiR ][ 61 0.577450 The Orphan [720] ][ 65 0.590498 Crossing Over 720P ][ 69 7.154971 Night At The Museum 2 [720p] ][ 71 2.696676 Elektra 720p ][ 73 6.929544 Underworld Series ][ 74 6.336581 The Lion King [1994] [720p] ][ 78 1.601014 Lord Of The Ring 1 - Fellowship of the Ring (ExtEd) [1080p] ][ 79 4.030397 Equilibrium 720p ][ 82 6.525550 Armageddon [1998] [720p] ][ 87 8.462529 Back.to.the.Future.Part.III ][ 89 3.984421 The.Godfather.Part.III.1990.720p.BluRay.x264-SiNNERS_tehPARADOX.COM ][ 90 5.320547 James Bond 09 - The Man with the Golden Gun [1974] ][ 91 7.080348 Indiana.Jones.and.the.Kingdom.of.the.Crystal.Skull    1080p ]
